File: Chapter 02/Python programs/2_10.py
no_my_toffee = int(input("Enter number of your toffees "))
no_sis_choco = int(input("Enter number of sister's chocolates "))
my_rs = 2 *no_my_toffee
sis_rs = 10 * no_sis_choco
if my_rs>sis_rs:
    print("I spent more money")
elif my_rs==sis_rs:
    print("We spent equal money")
else:
    print("She spent more money")

File: Chapter 02/C Programs/2_10.c
#include<stdio.h>
void main()
{
int no_my_toffee, no_sis_choco, my_rs, sis_rs;
printf("Enter No of your toffees ");
scanf("%d", &no_my_toffee);
printf("Enter no of sister toffee ");
scanf("%d", &no_sis_choco);
my_rs = 2 *no_my_toffee;
sis_rs = 10 * no_sis_choco;
if(my_rs>sis_rs)
printf("I spent more money");
else if (my_rs==sis_rs)
printf("We spent equal money");
else
printf("She spent more money");
}
